system/views/customer_view.py

```diff
@@ -13,18 +14,18 @@ def __init__(self, root, controller):
 
         # Frame labels
         product_label = ttk.Label(self.root, text="Choose a product")
-        product_label.grid(row=0, column=0, padx=10, pady=10)
+        product_label.grid(row=1, column=0, padx=10, pady=5)
         product_quantity_label = ttk.Label(self.root, text="Quantity")
-        product_quantity_label.grid(row=0, column=2, padx=10, pady=10)
+        product_quantity_label.grid(row=1, column=2, padx=10, pady=10)
 
         # Frame combobox/entries
         product_list = tk.StringVar()
         product_combobox = ttk.Combobox(self.root, width=34, state="readonly",
                                         textvariable=product_list)
         product_combobox["values"] = ("Laptop", "Monitors", "Mouse", "Keyboard")
-        product_combobox.grid(row=0, column=1, pady=10)
+        product_combobox.grid(row=1, column=1, pady=10)
         product_quantity_entry = ttk.Entry(self.root, width=18)
-        product_quantity_entry.grid(row=0, column=3, padx=10, pady=10)
+        product_quantity_entry.grid(row=1, column=3, padx=10, pady=10)
```

Comment:
You need to revert all the row/padding changes that you made when you edited CustomerView.
It seems like you added +1 to all the rows of the widgets. product_label.grid padding should be reverted to 10 also.